src/ui/views/Dashboard/Header.tsx

```diff
@@ -264,7 +264,7 @@ const Header = ({ loading }) => {
 
       await usewallet.lockWallet();
       await usewallet.clearWallet();
-      await usewallet.refreshAll();
+      // await usewallet.refreshAll();
```

Comment:
why we comment out `refreshAll` in here?

Reply:
because the reloading function has been updated, and we don't need to refresh everything before switching accounts now.

Reply:
If so, could we just delete this line ?